### ISSUE TITLE
Different Results

### ISSUE THREAD

**Issue by John-Almardeny** [stevenang/randomness_testsuite#3]
First of all, thanks very much for such amazing program with intuitive UI.

I am using your program and I want to include it in my research paper I'm writing.

One thing is confusing me is that: there are many different versions of NIST Test Suite implementation in Python and each gives different results (drastically different and not different in a few decimals that don't affect the final conclusion/result). I am really concerned about the reliability.

For example: [This](https://github.com/dj-on-github/sp800_22_tests) famous NIST Test Suite in Python fails all PRNGs regardless, while yours seems more realistic (fails some and pass others).

Any suggestion, comment or advice?

Many Thanks,
Yahya.

**Comment by stevenang** [comment]
Yahya,

Good day.

First of all, thanks for your comments.  I am really happy that my program can help you.

With regards your question, actually that's bother my a lot before and that is the motivation why I developed this program.

I cannot comment on what others are doing.  But on my side, I did the validation of the result generated by the program using the original NIST document (https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-22r1a.pdf).  The result of my program matched what is written in the document, and even found some error with the document.

With regards the differences of the result, it might be caused by the implementations and the library used by the developer. (This is my guess)

Thank you and good day.

Sincerely yours,

Steven Ang
